### ISSUE TITLE
Root volume IOPS incorrect for etcd nodes

### ISSUE THREAD

**Issue by bison** [coreos/tectonic-installer#1255]
## Versions

* Tectonic version: 1.6.6-tectonic.1-rc.2
* Terraform version: v0.9.9
* Platform: AWS

## What happened?
The change in #1166 is only correct for the default volume size. With a `gp2` volume size of 100 GB, the default IOPS will be 300 and a diff will still exist in the Terraform plan.

The real fix is to not pass the `iops` parameter at all for non `io1` volumes, but that doesn't work when everything is parameterized. You could also calculate the correct IOPS value for `gp2` volumes with:

```hcl
iops = "${var.root_volume_type == "io1" ? var.root_volume_iops : (3 * var.root_volume_size) + (100 % var.root_volume_size)}"
```

But that seems like a hack, and doesn't work for volumes other than `gp2` and `io1`.

Another option is to remove the ability to choose a volume type and IOPS for dedicated etcd clusters, and just make that decision for the user.

/cc @Quentin-M @alexsomesan

**Comment by Quentin-M** [comment]
I see, I was not aware of that, thanks.

**Comment by sym3tri** [comment]
Moved to Jira https://coreosdev.atlassian.net/browse/INST-189

**Comment by squat** [comment]
As documented in https://github.com/coreos/tectonic-installer/pull/1826#issuecomment-327951707, this issue is solved by #1826 and #953. The final behavior is:
* if volume type is IO1 -> let user set IOPS
* if volume type is GP2 -> set ignore IOPS and set IOPS to size*3
* if volume type is anything else, set IOPS to 0.
Please reference the implementation here: https://github.com/coreos/tectonic-installer/blob/master/modules/aws/etcd/nodes.tf#L51
